core/src/tangram.cpp

```diff
@@ -196,7 +196,7 @@ void update(float _dt) {
             m_labels->update(*m_view, _dt, m_scene->styles(), tiles, cache);
         }
 
-        bool animated = false;
+        bool animated = m_scene->animated();
         for (const auto& style : m_scene->styles()) {
             if (style->isAnimated()) {
```

Comment:
Based on the discussion on #617, it seems if `scene.animated = false` then scene will not be animated irrespective of the value of `style.animate`.
However here, style animate will override scene animate property.